azuracast/liquidsoap_custom.liq

```diff
@@ -359,9 +368,8 @@ sources = [
     ({5w and 12h30m-21h30m}, playlist_off_air_ambient),                        #ambient: Ritka pentek - Boombap
     ({6w and 11h30m-13h28m}, once(playlist_merites)),                  #Saturday
     ({6w and 13h30m-14h58m}, once(playlist_kerodzo)),     
-    ({6w and 15h-17h58m}, once(playlist_torso)),
-    ({6w and 18h-19h58m}, once(playlist_rnr666)),
-    ({6w and 20h-21h58m}, once(playlist_mood_sequence)),
+    ({6w and 15h-15h58m}, once(playlist_torso)),
+    ({6w and 16h-22h58m}, once(playlist_lahma_kripta_support)),
```

Comment:
We don't need a playlist live. Since we'll never play it back, it's an unnecessary complication.